### ISSUE TITLE
Tunnels howto references non-existant ../examples/tuntap.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6898]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)| @dreid reported|
|-|-|
|Trac ID|trac#6898|
|Type|defect|
|Created|2014-01-14 18:37:58Z|
|Branch|https://github.com/twisted/twisted/tree/tuntap-example-6898|

It looks like this should have been included in [#6169](https://github.com/twisted/twisted/issues/6169) but wasn't.

https://twistedmatrix.com/trac/changeset/41202

<details><summary>Searchable metadata</summary>

```
trac-id__6898 6898
type__defect defect
reporter__dreid dreid
priority__normal normal
milestone__ 
branch__branches_tuntap_example_6898 branches/tuntap-example-6898
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__pair pair
keywords__documentation documentation
time__1389724678000000 1389724678000000
changetime__1389873649000000 1389873649000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

It's just a bad link to the `pairudp.py` example.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [41266]) Branching to 'tuntap-example-6898'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

I changed the link.  I also cleaned up the crazy whitespace.

Due to [#6903](https://github.com/twisted/twisted/issues/6903) I don't know if this actually produces a working link, nor whether the whitespace changes are actually correct.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek commented|
|-|-|


 1. It builds and looks fine on Sphinx 1.2
 1. The link works too.
 1. Shouldn’t it be “in **the** XYZ example”? Sounds odd to me, but this may be German/Czech bias of mine.
 1. The relative links looks funny, wouldn’t :download:`pairudp.py <../examples/pairudp.py>` be nicer?

Both 3. and 4. are up to your own judgement, otherwise please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Thanks.  Fixed (3) in r41269.  I'm not familiar with that reST syntax off-hand and I still can't build the docs myself so I'd rather not try making that change.  It might be a nice improvement for later though.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [41271]) Merge tuntap-example-6898

Author: exarkun
Reviewer: hynek
Fixes: [#6898](https://github.com/twisted/twisted/issues/6898)

Fix an incorrect link to an example in the tunnels howto.  Also remove all of
the excessive tool-generated whitespace from the XHTML to reST conversion.
